### PR TITLE
terminationGracePeriodSeconds: stable in 1.27

### PR DESCRIPTION
**Pull request by rphillips**
Bumps terminationGracePeriodSeconds to stable for 1.27.

Reference:  https://github.com/kubernetes/enhancements/issues/2238

cc @mickeyboxell

### CODE REVIEW

**Comment by Rishit-dagli** [comment]
Hello @rphillips  👋 
I’m reaching out from the 1.27 Release Docs team. This docs PR should be made against the [`dev-1.27` branch](https://github.com/kubernetes/website/tree/dev-1.27).

Also, the Docs PRs reviewed and are ready to merge deadline is 4th April 2023 and this PR lacks a review.

/sig node

**Review by Rishit-dagli — APPROVED** [review]
Looks like this does not need any other documentation updates /lgtm

PS: Before merging this needs to still point to dev-1.27

**Comment by rphillips** [comment]
@Rishit-dagli : Thanks... Updated the base

**Review by mrunalp — APPROVED** [review]

**Comment by reylejano** [comment]
All related PRs merged for 1.27, see comment in https://github.com/kubernetes/enhancements/issues/2238#issuecomment-1462179404

/cc @mickeyboxell 
/lgtm

**Pull request merged**
